execution/execution.proto

```diff
@@ -144,6 +144,7 @@ message AssembledBlockData {
     types.ExecutionPayload execution_payload = 1;
     types.H256 block_value = 2;
     types.BlobsBundleV1 blobs_bundle = 3;
+    types.H256 parent_beacon_block_root = 4;
```

Comment:
Should this be optional?

Reply:
no, it should be require

Reply:
Why

Reply:
It must be require: `parentBeaconBlockRoot` can be nil(optional). Ref: https://github.com/ethereum-optimism/optimism/blob/develop/specs/exec-engine.md#engine_getpayloadv3